jaeger/charts/linkerd-jaeger/values.yaml

```diff
@@ -285,10 +285,11 @@ webhook:
   logLevel: info
 
   namespaceSelector:
-    #matchExpressions:
-    #- key: runlevel
-    #  operator: NotIn
-    #  values: ["0","1"]
+    matchExpressions:
+    - key: kubernetes.io/metadata.name
+      operator: NotIn
+      values:
+      - kube-system
```

Comment:
I think adding a doc comment just above `namespaceSelector`  (like in the `linkerd-control-plane` chart) will avoid `bin/helm-doc` generating undesired entries for each one of these entries.